hooks/boot/13-tuning.rb

```diff
@@ -4,10 +4,11 @@
   TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
   TUNING_FACT = 'tuning'.freeze
 
-  app_option(
-    '--tuning',
-    'INSTALLATION_SIZE',
-    "Tune for an installation size. Choices: #{TUNING_SIZES.join(', ')}",
-    :default => get_custom_fact(TUNING_FACT)
-  )
+  app_option('--tuning', 'INSTALLATION_SIZE', "Tune for an installation size. Choices: #{TUNING_SIZES.join(', ')}") do
+    unless TUNING_SIZES.include?(value)
+      puts "Invalid option supplied for --tuning. Please choose from one of: #{TUNING_SIZES.join(', ')}"
+      puts "See: 'foreman-installer --help'"
+      exit 101
+    end
```

Comment:
I think you need to return the value as well per https://github.com/mdub/clamp#validation
```suggestion
    end
  value
```